src/app/planner/planner/planner.component.html

```diff
@@ -1,14 +1,13 @@
 <div class="planner-container">
-	<div class="card" *ngFor="let item of meals; let myIndex=index">
-		<div class="card-info">
-		<img class="meal-img" src='{{item.url}}' alt="meal">
-		<h2>{{item.name}}</h2>
-		</div>
-	  <div class="card-portion">
-		<h3>{{item.portion}}</h3>
-			<button id="remove-button" (click)="deleteMeals(item.id)"><h3>X</h3></button>
-		</div>
-	</div>
+	<app-card 
+	*ngFor="let item of meals; let myIndex=index" 
+	[id]="item.id" 
+	[name]="item.name" 
+	[portion]="item.portion" 
+	[url]="item.url"
+	(delete)="deleteMeals(item.id)">{{item.name}}
+	</app-card>
 	<button id="add-button" (click)="updateMeals()"><h3>+</h3></button>
```

Comment:
You have a reusable button. It would be better if you can make it flexible enough that you can reuse it here as well. 

Tips:
- use <ng-content> to make the button content flexible. You can then use your button like this: <app-button>X</app-button>
- give your button a prop that can change the width. You can use ngClass.